wiki/Manual.md

```diff
@@ -1264,9 +1264,7 @@ Public APIs include the following:
 - OHLCV series for charting
 - other public endpoints
 
-For trading with private API you need to obtain API keys from/to exchanges. It often means registering with exchanges and creating API keys with your account. Most exchanges require personal info or identification. Some kind of verification may be necessary as well.
-
-If you want to trade you need to register yourself, this library will not create accounts or API keys for you. Some exchange APIs expose interface methods for registering an account from within the code itself, but most of exchanges don't. You have to sign up and create API keys with their websites.
+A private API is mostly used to access account-specific, personal data and require authentication. To use private API you need to obtain API keys from/to exchanges. For that, in most cases, you will need to register with exchanges and create API keys with your account ( CCXT will not create accounts or API keys for you ). Most exchanges require personal info or identification, or sometimes even a verification.
```

Comment:
On this paragraph, tbh, i would keep most of it as it was, and would probably remove the reference to creating the API keys from code. I would suggest something like:

> The private API is mostly used to access account-specific private data and requires authentication. You have to get the private API keys from the exchanges. It often means registering with an exchange website and creating the API keys for your account. Most exchanges require personal information or identification. Some exchanges will only allow trading after completing the KYC verification.

On a sidenote – I noticed that you have a habit of putting a space after the opening bracket `(`. Let's stick to the natural syntax, do not add spaces after opening round brackets and before closing round brackets in text or in code. Taking it further, the less punctuation, the more readable and understandable the docs are. The good rule of thumb: if you often feel like you would put some clarification in round brackets, that can be re-phrased as just one more sentence without brackets, just to keep the text cleaner.

Reply:
good notes. I'll follow that.